deploy/base/apps/monitoring/prometheus/configMap/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+configMapGenerator: # This generates prometheus configMap from prometheus.yml
+  - name: prometheus-server-conf
+    namespace: monitoring
+    files:
+    - prometheus.yml # The .data in yaml will have the same key/tag as file name
+    options:
+      # TODO: Will it add a suffix everytime there are new changes to configmap??
```

Comment:
@AAkindele Thanks for catching this.
I've already tested this statement.
It seems the hash is generated based on file content (maybe a combination of md5, crc and sha).

It will change the suffix everytime there is a change in file (the actual config file change, not the configMap yaml).

If there is not change, it will not change the hash